packages/superset-ui-number-format/src/NumberFormatterRegistry.ts

```diff
@@ -14,6 +15,9 @@ export default class NumberFormatterRegistry extends RegistryWithDefaultKey<
       overwritePolicy: OverwritePolicy.WARN,
     });
 
+    this.registerValue(NumberFormats.DURATION, createDurationFormatter());
```

Comment:
Please revert the changes to this file. The duration formatter should be optional and not registered by default. There are other apps using this package and some do not need duration formatter. You can register these formatters in `incubator-superset`'s `setupFormatter.js`.  That way other apps can take advantage of tree-shaking and won't have to bundle `pretty-ms` when they are not using.

Reply:
You can add a section in `README` to instruct how to make duration formatter available.

Reply:
Ok that makes sense. I wasn't able to find `setupFormatter.js` or any reference to `getNumberFormatterRegistry()` in `incubator-superset`, but I think I have a good idea of how it should be done. So if it's ok I'll open a new PR to add instructions to `README.md` once `0.12` is released and `durationFormatter` gets implemented in the backend.

Reply:
Sorry I made a typo. It is `setupFormatters.js`. For `README` I meant `packages/superset-ui-number-format/README.md`